.dockerignore

```diff
@@ -4,13 +4,8 @@ fly.toml
 tmp
 !tmp/pids
 log
-public/assets
-public/packs
```

Comment:
NIT: Actually, I think it's probably fine to just leave them in `.dockerignore`. Just in case in the future we somehow really produce something in `public/(assets|packs)` during any step of docker file, we probably would bundle them into the final image without noticing.

Reply:
[fixed](https://github.com/qoosuperman/taverneer-backend/pull/17/commits/fceb4b7b6b470cbd3d12bafc1ea526beb9da01d3#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5L10-L12)